version.gradle.kts

```diff
@@ -32,4 +32,4 @@
  *
  * For dependencies on Spine SDK module please see [io.spine.dependency.local.Spine].
  */
-val protoDataVersion: String by extra("0.69.7")
+val protoDataVersion: String by extra("0.69.8")
```

Comment:
If the API has any breaking changes (has it? considering the changes in `options.proto`?), this should be `0.70.0`.